### DISCUSSION TITLE
Bricked Huawei P20 Lite, no clues

### DISCUSSION THREAD

**Discussion by CtrlValCanc** [kitsuned/PotatoNV#152] [Q&A]
I can access fastboot, erecovery and recovery. It's like it can flash twrp but it reboots into recovery. 
I tried redoing the bootloader unlock to see if it can change anything, but now if I select 65x (A) it gives me _ACK is invalid! ACK=55; Excepted=AA_, but the bootloader should be correct.

also, i would like to disable fblock.

**Comment by werasik2aa** [comment]
I can try to help in recovering it
